migrations/20141121173412_add_talk_meta.php

```diff
@@ -15,8 +15,8 @@ public function change()
             ->addColumn('rating', 'integer', ['default' => 0])
             ->addColumn('viewed', 'boolean', ['default' => false])
             ->addColumn('created', 'datetime')
-            ->addIndex(['admin_user_id', 'talk_id'], ['name' => 'talk_meta--admin_user_id__talk_id', 'unique' => true])
+            ->addIndex(['admin_user_id', 'talk_id'], ['name' => 'talk_meta__admin_user_id__talk_id', 'unique' => true])
```

Comment:
For databases that supported the `--` characters, do we want to fix the index names or leave it alone?

Reply:
I changed them because Postgres doesn't like them